README.md

```diff
@@ -1,44 +1,45 @@
-# SDR Deployment script
+# SDR Deployment Tools
 
-This is a central place for deploying SDR applications (and sul_pub).  This allows all applications
-to be deployed together in a single script.
+This is a central place for deploying SDR applications (and sul_pub). This allows all applications
+to be deployed together with a single set of tools.
 
 ## Usage
 
 Make sure that:
+
 * You are on VPN.
 * You have `kinit`-ed.
 * You have added the public SSH key, often `~/.ssh/id_rsa.pub`, from your machine to [GitHub](https://github.com/settings/keys)
 * You have previously `ssh`-ed into all servers.
-  * NOTE: If you are unsure about this, run `./deploy.rb [qa|stage|prod] --checkssh` and watch the output for any errors!
+  * NOTE: If you are unsure about this, run `bin/check_ssh -e [qa|stage|prod]` and watch the output for any errors!
 
-### Check your ssh connection to all servers
+### Check your SSH connection to all servers
 
 ```
-./deploy.rb [qa|stage|prod] --checkssh
+bin/check_ssh -e [qa|stage|prod]
 ```
 
 NOTE: Watch the output for any errors
 
-### Check versions of cocina
+### Check versions of cocina-models
 
 ```
-./deploy.rb [qa|stage|prod] --check-cocina
+bin/check_cocina
```

Comment:
Test to see what happens when these commands are invoked via bx and tweak docs if necessary

Reply:
`bundle exec` does not affect the behavior.